### PR TITLE
Implemented Account API

### PR DESCRIPTION
**Pull request by srs-vimalk**
**Account**
Create Account
Create Multiple Accounts
Read Account
List Accounts
Find Matching Accounts
Update Account
Add Note to Account
Delete Account

**Link / Unlink Account to Contacts**
Link Account to Contact
Link Account to Multiple Contacts
Unlink an Account With a Contact
Account Classifications

**Create Classification**
Read Classification
List All Classifications
Update Classification
Delete Classification

**Account Types**
Create Type
Read Type
List All Types
Update Type
Delete Type

### CODE REVIEW

File: LiveVox.NET/LiveVox.NET.csproj [hunk 1]
```diff
@@ -7,6 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```
Comment (srs-adamr) [inline LiveVox.NET/LiveVox.NET.csproj]: Remove this, we are using system.Text.Json

File: LiveVox.NET/Models/Account/Requests/LinkAccountToContactRequest.cs
Comment (srs-adamr) [inline LiveVox.NET/Models/Account/Requests/LinkAccountToContactRequest.cs:1]: Missing Url Segment

File: LiveVox.NET/Models/Account/Requests/ListAllAccountTypeRequest.cs
Comment (srs-adamr) [inline LiveVox.NET/Models/Account/Requests/ListAllAccountTypeRequest.cs:1]: This looks incomplete.

File: LiveVox.NET/Models/Account/Requests/ListAllClassificationRequest.cs
Comment (srs-adamr) [inline LiveVox.NET/Models/Account/Requests/ListAllClassificationRequest.cs:1]: This looks incomplete

File: LiveVox.NET/Models/Account/Requests/ReadAccountRequest.cs
Comment (srs-adamr) [inline LiveVox.NET/Models/Account/Requests/ReadAccountRequest.cs:1]: This looks incomplete.

File: LiveVox.NET/Models/Account/Requests/ReadAccountTypeRequest.cs
Comment (srs-adamr) [inline LiveVox.NET/Models/Account/Requests/ReadAccountTypeRequest.cs:1]: This looks incomplete.

File: LiveVox.NET/Models/Account/Requests/ReadClassificationRequest.cs
Comment (srs-adamr) [inline LiveVox.NET/Models/Account/Requests/ReadClassificationRequest.cs:1]: This looks incomplete.

File: LiveVox.NET/Models/Account/Responses/ListAllClassificationResponse.cs
Comment (srs-adamr) [inline LiveVox.NET/Models/Account/Responses/ListAllClassificationResponse.cs:1]: This is missing Json attributes

**Review by srs-adamr — CHANGES_REQUESTED** [review]
Please review comments

**Review by srs-adamr — APPROVED** [review]

File: LiveVox.NET/Models/Account/Common/Account.cs [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+        /// Gets or sets the primary account value.
+        /// </summary>
+        [JsonPropertyName("primaryAccountContact")]
+        public string PrimaryAccountContact { get; set; }
+
+        /// <summary>
+        /// Gets or sets the error ID.
+        /// </summary>
+        [JsonPropertyName("logId")]
+        public Guid LogId { get; set; }
```
Comment (srs-binnys) [inline LiveVox.NET/Models/Account/Common/Account.cs]: @srs-vimalk pls check if we need Guid's or if we can have string instead.

**Review by srs-binnys — APPROVED** [review]

**Pull request merged**